0002-chunking.md

```diff
@@ -0,0 +1,271 @@
+## Summary
```

Comment:
```suggestion
## Chunking summary
```

Reply:
I suggest we change "Summary" to the gist of the RFC so we can identify the subject matter at a glance, otherwise, all of our RFCs will be titled "Summary" (not very useful)

Reply:
@bkase wdyt for the template? Should we propose a structure that customises the summary for each?

Reply:
We could add a top-level `#` heading with the title of the feature maybe? But then keep `Summary` because it's a standard. What do you think @barriebyron ?

Reply:
better, yes. I updated the template, too